programs/openbook-v2/src/state/open_orders_account.rs

```diff
@@ -9,7 +9,7 @@ use crate::pubkey_option::NonZeroPubkeyOption;
 
 use super::{BookSideOrderTree, FillEvent, LeafNode, Market, Side, SideAndOrderTree};
 
-pub const MAX_OPEN_ORDERS: usize = 128;
+pub const MAX_OPEN_ORDERS: usize = 24;
```

Comment:
is this 24 or 32? (the git comment says 32)

Reply:
24